src/Decoder.cpp

```diff
@@ -731,7 +731,19 @@ size_t BitpackIntegerDecoder<RegisterT>::inputProcessAligned( const char *inbuf,
 #endif
 
       RegisterT w;
-      if ( bitOffset > 0 )
+      if ( bitOffset == 0 )
+      {
+         /// The left shift (used below) is not defined if shift is >= size of
+         /// word
+         w = low;
+      }
+      // Avoid reading the next word, unless it is needed
+      // If the last record finishes on the last bit of input, avoid UMR
+      else if ( bitOffset + bitsPerRecord_ <= registerBits_)
```

Comment:
Missing space. Could you please run clang-format to make sure everything's proper? (see CONTRIBUTING.md)

Reply:
Yes, done.